test/track.test.js

```diff
@@ -253,6 +265,10 @@ describe('ravelin.track', function() {
 
         r = new Ravelin(isolate({key: key, api: '/'}));
 
+        if (test.cfg && test.cfg.classifyPaste) {
+          r.classifyPaste = test.cfg.classifyPaste;
```

Comment:
I think we'd want to set r.track.classifyPaste - but lets instead try to build the constructor args. jQuery's got a handy extend function for it:

```js
r = new Ravelin(isolate($.extend({}, test.cfg, {key: key, api: '/'));
```

Reply:
Ooh that's cool, I didn't know of a non es6 way to do this